<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuickFitDialog</class>
 <widget class="QDialog" name="QuickFitDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>快速分配</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>310</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>作业大小：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bindButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>320</y>
     <width>110</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>分配</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>320</y>
     <width>110</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
